src/parser.ts

```diff
@@ -78,8 +81,8 @@ export class Parser {
         }
     }
 
-    latexmkSkipped(log: string) : boolean {
-        const lines = log.replace(/(\r\n)|\r/g, '\n').split('\n')
+    latexmkSkipped(log: string): boolean {
+        const lines = log.split('\n')
```

Comment:
I'm not an expert in LaTeX logging system. Does the compiler return `\n` instead of `\r\n` in Windows environment? An universal `\n` is highly likely, though.